plugins/analytics-module-segment/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@internal/analytics-module-segment",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin-module"
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "diff": "backstage-cli plugin:diff",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@backstage/catalog-model": "^1.0.1",
+    "@backstage/config": "^1.0.1",
+    "@backstage/core-components": "^0.11.0",
+    "@backstage/core-plugin-api": "^1.0.5",
+    "@backstage/theme": "^0.2.16",
```

Comment:
Note that all `@backstage` deps need to be updated before merging. Bunch of extra `yarn.lock` changes because of it atm